docs/books/admin_guide/03-commands.md

```diff
@@ -35,42 +35,42 @@ Current Linux systems have graphical utilities dedicated to the work of an admin
+* A **command prompt**, or __prompt__ login, which is symbolized by a `#` for administrators and a `$` for other users (`/etc/profile`).
 
-Depending on the security policy implemented on the system, the password will have to contain a certain number of characters and meet certain complexity requirements.
+Depending on the security policy implemented on the system, the password will require a certain number of characters and meet certain complexity requirements.
 
-Among the existing command interpreters, the **Bourne-Again Shell** (`/bin/bash`) is the one most frequently used. It is assigned by default to new users. For various reasons, advanced Linux users can choose alternative shells from among the Korn Shell (`ksh`), the C Shell (`csh`), etc.
+Among the existing command interpreters, the **Bourne-Again Shell** (`/bin/bash`) is the one most frequently used. It is assigned by default to new users. For various reasons, advanced Linux users can choose alternative shells such as the Korn Shell (`ksh`), the C Shell (`csh`), etc.
 
 The user's login directory is by convention stored in the `/home` directory of the workstation. It will contain the user's personal data and the configuration files of his applications. By default, at login, the login directory is selected as the current directory.
 
-A workstation type installation (with graphical interface) starts this interface on terminal 1. Linux being multi-user, it is possible to connect several users several times, on different **physical terminals** (TTY) or **virtual terminals** (PTS). Virtual terminals are available within a graphical environment. A user switches from one physical terminal to another using <kbd>Alt</kbd> + <kbd>Fx</kbd> from the command line or using <kbd>CTRL</kbd> + <kbd>Alt</kbd> + <kbd>Fx</kbd>.
+A workstation-type installation (with graphical interface) starts this interface on terminal 1. Linux being multi-user, it is possible to connect several users several times, on different **physical terminals** (TTY) or **virtual terminals** (PTS). Virtual terminals are available within a graphical environment. A user switches from one physical terminal to another using <kbd>Alt</kbd> + <kbd>Fx</kbd> from the command line or using <kbd>CTRL</kbd> + <kbd>Alt</kbd> + <kbd>Fx</kbd>.
 
 ### The shell
 
-Once the user is connected to a console, the shell displays the command **prompt**. It then behaves like an infinite loop, with each statement entered:
+Once the user is connected to a console, the shell displays the **command prompt**. It then behaves like an infinite loop, repeating the same pattern with each statement entered:
 
-* displaying the command prompt;
-* reading the command;
-* syntax analysis;
-* substitution of special characters;
-* execute the command;
-* display the command prompt;
-* etc.
+* Display the command prompt.
+* Reading the command.
+* Analyze syntax.
+* Substitute special characters.
+* Execute the command.
+* Display the command prompt.
+* Etc.
```

Comment:
@Grammaresque great work! I'm wondering if the these should be changed to plurals. This is just a suggestion:
Displays the command prompt.
Reads the command.
Analyzes the syntax.
Substitutes special characters.
Executes the command.
Displays the command prompt.

Thoughts?

Reply:
Yes! I was trying to make them all consistent in that present tense and I did miss this list. Am I able to fix that or do you do it now that I've already submitted?